webapp/pages/profile/_id/_slug.vue

```diff
@@ -68,16 +68,23 @@
           </ds-flex>
           <ds-space margin="small">
             <template v-if="!myProfile">
+              <base-button
+                v-if="user.isBlocked"
+                @click="unblockUser(user)"
+                class="unblock-user-button"
+              >
+                {{ $t('settings.blocked-users.unblock') }}
+              </base-button>
+              <base-button v-if="user.isMuted" @click="unmuteUser(user)" class="unmute-user-button">
+                {{ $t('settings.muted-users.unmute') }}
+              </base-button>
```

Comment:
It's possible that you could see both unblock and unmute buttons, right? I wonder how that would look like :thinking:

Reply:
I uploaded a screen shot, but somehow I added it as a review, and cancelled it, without knowing it would remove it.

![Screenshot from 2020-01-23 19-23-50](https://user-images.githubusercontent.com/26943915/73342473-edbb2980-427e-11ea-90e0-cd21562f6134.png)